utils/io/fileutils/files.go

```diff
@@ -484,3 +484,15 @@ func FindUpstream(itemToFInd string, itemType ItemType) (wd string, exists bool,
 }
 
 type ItemType string
+
+func FilesIdentical(src string, toCompare string) (bool, error) {
```

Comment:
Let's add the following description for the function:
// Returns tru if the two files have the same MD5 checksum.

I suggest renaming the two arguments to file1 and file2.